frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.ts

```diff
@@ -206,6 +218,31 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>({
                     groupTypeIndex: index,
                 })),
         ],
+        infiniteListLogics: [
+            (s) => [s.taxonomicGroupTypes, (_, props) => props],
+            (taxonomicGroupTypes, props): Record<string, BuiltLogic<infiniteListLogicType>> =>
+                Object.fromEntries(
+                    taxonomicGroupTypes.map((groupType) => [
+                        groupType,
+                        infiniteListLogic.build({
+                            ...props,
+                            listGroupType: groupType,
+                        }),
+                    ])
+                ),
+        ],
+        totalCounts: [
+            (s) => [
+                (state, props) =>
```

Comment:
This is pretty cool, didn't know you could do this in kea

Reply:
This is a bit of a hack --> the counts are pretty much re-rendered every time this selector is read... but since this is just just reading 4+ (memoized) selectors, the recalculation itself won't be noticeable. Still we should be aware of the situation when using this style of selectors.